app/views/comments/_comment.erb

```diff
@@ -0,0 +1,35 @@
+<% comment||=@comment %>
```

Comment:
sau các dấu`,` và các toán tử thì cần phải có 1 khoảng trắng ví dụ `comment ||= @comment`
em check kỹ lại toàn bộ code vì a tháy có nhiều chỗ phải sửa cái này

Reply:
a vẫn chưa hiểu sao lại cần gán comment là @comment nhỉ 
sao các chỗ bên dưới ko dùng @comment luôn